cdragontoolbox/binfile.py

```diff
@@ -51,7 +52,7 @@ def __eq__(self, other):
     def __str__(self):
         if self.s is not None:
             return self.s
-        return f"{{{self.h:08x}}}"
+        return self.hex
```

Comment:
This changes the output (it removes surrounding braces), which is already used by other parts of the code.